Autoupdater fails on v20.2369 without log or trace. Manually starting newer jackett produces segmentation fault

### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Environment version: 6.0.11 (/home/apps/jackett/Jackett/)
OS version: Unix 5.15.74.64 (64bit OS) (64bit process)
Jackett variant: CoreLinuxArm64
File /etc/issue: Armbian 22.11.1 Focal \l
Running in Docker: No
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
App config/log directory: /home/apps/.config/Jackett
Using proxy: Disabled
Using FlareSolverr: No
Using HTTP Client: HttpWebClient2
```


### Description

Hi,

auto-updater was killing my jackett instance with no traces nor log, so I manually installed and started any released version newer than .20.2369 and they all fail with segmentation fault. Even adding -t or -l doesn't really help as it seems that the binary is not even executing.

My platform is ARM64 running Armbian.

Rolling back to v20.2369 works again, but I had to disable autoupdate due to the recent versions not updating anymore.
I will stay on 2369 until you need any help on the topic, but obviously feel free to ask for any info you might need.

I tried to delete .config/jackett dir, but it fails on the same way (segfault)

Thanks in advance!

### Logged Error Messages

```text
As I said, no logs get in log.txt nor update.txt
```


### Screenshots

_No response_

Mono version also fails, hope that my mono version is ok and the trace gives you some hint:

```
myprompt:~/Jackett$ mono ./JackettConsole.exe
12-27 13:20:53 Info Starting Jackett v0.20.2444
12-27 13:20:54 Error System.TypeLoadException: Could not resolve type with token 010000a1 from typeref (expected class 'System.Runtime.InteropServices.OSPlatform' in assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51')
  at Microsoft.AspNetCore.WebHost.CreateDefaultBuilder (System.String[] args) [0x000d3] in <96ee8d1b889d4f26b7bd4cc3f07ca395>:0
  at Jackett.Server.Program.CreateWebHostBuilder (System.String[] args, System.String[] urls, System.String contentRoot) [0x00000] in <7fb70a666c614c63a7844a95a4e8eeea>:0
  at Jackett.Server.Program.Main (System.String[] args) [0x002b2] in <7fb70a666c614c63a7844a95a4e8eeea>:0  System.TypeLoadException: Could not resolve type with token 010000a1 from typeref (expected class 'System.Runtime.InteropServices.OSPlatform' in assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51')
  at Microsoft.AspNetCore.WebHost.CreateDefaultBuilder (System.String[] args) [0x000d3] in <96ee8d1b889d4f26b7bd4cc3f07ca395>:0
  at Jackett.Server.Program.CreateWebHostBuilder (System.String[] args, System.String[] urls, System.String contentRoot) [0x00000] in <7fb70a666c614c63a7844a95a4e8eeea>:0
  at Jackett.Server.Program.Main (System.String[] args) [0x002b2] in <7fb70a666c614c63a7844a95a4e8eeea>:0

Unhandled Exception:
System.TypeLoadException: Could not resolve type with token 010000a1 from typeref (expected class 'System.Runtime.InteropServices.OSPlatform' in assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51')
  at Microsoft.AspNetCore.WebHost.CreateDefaultBuilder (System.String[] args) [0x000d3] in <96ee8d1b889d4f26b7bd4cc3f07ca395>:0
  at Jackett.Server.Program.CreateWebHostBuilder (System.String[] args, System.String[] urls, System.String contentRoot) [0x00000] in <7fb70a666c614c63a7844a95a4e8eeea>:0
  at Jackett.Server.Program.Main (System.String[] args) [0x002fd] in <7fb70a666c614c63a7844a95a4e8eeea>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not resolve type with token 010000a1 from typeref (expected class 'System.Runtime.InteropServices.OSPlatform' in assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51')
  at Microsoft.AspNetCore.WebHost.CreateDefaultBuilder (System.String[] args) [0x000d3] in <96ee8d1b889d4f26b7bd4cc3f07ca395>:0
  at Jackett.Server.Program.CreateWebHostBuilder (System.String[] args, System.String[] urls, System.String contentRoot) [0x00000] in <7fb70a666c614c63a7844a95a4e8eeea>:0
  at Jackett.Server.Program.Main (System.String[] args) [0x002fd] in <7fb70a666c614c63a7844a95a4e8eeea>:0
myprompt:~/Jackett$ mono --version
Mono JIT compiler version 6.12.0.182 (tarball Tue Jun 14 22:41:13 UTC 2022)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       normal
        Notifications: epoll
        Architecture:  arm64
        Disabled:      none
        Misc:          softdebug
        Interpreter:   yes
        LLVM:          yes(610)
        Suspend:       preemptive
        GC:            sgen (concurrent by default)
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -7,7 +7,7 @@ variables:
   jackettVersion: $(majorVersion).$(minorVersion).$(patchVersion)
   buildConfiguration: Release
   netCoreFramework: net6.0
-  netCoreSdkVersion: 6.0.403 # #13806 & dotnet/runtime#79796
+  netCoreSdkVersion: 6.0.x
   # system.debug: true
 
 trigger:

